qemu/tests/virtio_fs_supp_group_transfer.py

```diff
@@ -0,0 +1,87 @@
+       and set the dir's group to 'wheel'
+    5) In guest, using root to create a test directory at the shared dir.
+    6) Run the basic io test with the u1 user
+    7) After test, clear the environment.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    add_user_cmd = params.get("add_user_cmd")
+    del_user_cmd = params.get("del_user_cmd")
+    username = params.get("new_guest_user")
+    fs_dest = params.get("fs_dest")
+    fs_target = params.get("fs_target")
+    fs_source_dir = params.get("fs_source_dir")
+
+    testdir = "testdir"
+    guest_root_session = None
+    guest_user_session = None
+
+    try:
+        shared_dir = params.get("fs_source_dir")
+
+        vm = env.get_vm(params.get("main_vm"))
+        vm.verify_alive()
+        guest_root_session = vm.wait_for_login()
+
+        error_context.context("Create a destination directory %s "
+                              "inside guest." % fs_dest, test.log.info)
+        if not utils_misc.make_dirs(fs_dest, session=guest_root_session):
+            test.fail("Creating directory was failed!")
+        error_context.context("Mount virtiofs target %s to %s inside"
+                              " guest." % (fs_target, fs_dest),
+                              test.log.info)
+        if not utils_disk.mount(fs_target, fs_dest, 'virtiofs',
+                                session=guest_root_session):
+            test.fail('Mount virtiofs target failed.')
+
+        guest_root_session.cmd(add_user_cmd % username)
+        guest_root_session.cmd("usermod -G wheel %s" % username)
+
+        guest_root_session.cmd("cd %s && mkdir -m 770 %s" % (fs_dest, testdir))
+        guest_root_session.cmd("cd %s && chgrp wheel %s" % (fs_dest, testdir))
+        params["fs_dest"] = os.path.join(fs_dest, testdir)
+        params["fs_source_dir"] = os.path.join(shared_dir, testdir)
+
+        try:
+            guest_user_session = vm.wait_for_login()
+            guest_user_session.cmd("su %s" % username)
+            virtio_fs_utils.basic_io_test(test, params, guest_user_session)
+        except Exception:
+            guest_user_session = None
+    finally:
+        params["fs_dest"] = fs_dest
+        params["fs_source_dir"] = fs_source_dir
+        if guest_user_session:
+            guest_user_session.close()
+        if guest_root_session:
+            output = guest_root_session.cmd_output(del_user_cmd % username)
+            if "is currently used by process" in output:
+                error_context.context("Kill process before delete user...",
+                                      test.log.info)
+                pid = output.split(" ")[-1]
+                guest_root_session.cmd_output("kill -9 %s" % pid)
+            guest_root_session.cmd("rm -rf /home/%s" % username)
+
+            guest_root_session.cmd("rm -rf %s" % params["fs_dest"])
```

Comment:
we set fs_dest back to /mnt/myfs at line72, note here, this is the mount point which mount myfs into guest os, and currently, the guest os is still running and we didn't umount the myfs, then we remove the mount point directly, is that expected?